hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/UtilVerbs.java

```diff
@@ -731,7 +730,9 @@ public static NewSpecKeyList newKeyListNamed(String key, List<String> childKeys)
      * @return the operation that runs the sub-operations in parallel
      */
     public static GroupedOps<?> inParallel(@NonNull final SpecOperation... subs) {
-        return repeatableModeRequested() ? blockingOrder(subs) : new ParallelSpecOps(subs);
+        return "repeatable".equalsIgnoreCase(System.getProperty("hapi.spec.embedded.mode"))
+                ? blockingOrder(subs)
+                : new ParallelSpecOps(subs);
```

Comment:
```suggestion
        final var isRepeatable = "repeatable".equalsIgnoreCase(System.getProperty("hapi.spec.embedded.mode"));
        if (isRepeatable) {
                return blockingOrder(new LogMessage("WARN: Parallel execution not compatible with 'repeatable'; running ops in sequential order"), subs);
        } else {
            return new ParallelSpecOps(subs);
        }
```

Reply:
This one I don't see much reason to log anything, since there's no functional deviation caused by running sequentially.

Reply:
Merely a warning to the reader or caller about expectations–I didn't know `inParallel()` would silently ignore a request for parallel execution based on the mode. Maybe I'll add something to the javadoc in a future PR somewhere